Game_Manual/Stats.md

```diff
@@ -29,31 +29,83 @@ How much your team members felt that you supported them in their learning. This
-| #red-rabbits | 28%                           | 30%                           |
+| Project      | Team size | Self estimate | Team estimates |
+|:-------------|:----------|:--------------|:---------------|
+| #big-bees    | 3         | 50%           | 42%, 48%       |
+| #red-rabbits | 4         | 28%           | 27%, 35%, 28%  |
+| #tiny-tigers | 2         | 20%           | 30%            |
 
-Your discernment _per project_ is the difference between self- and team-estimated contribution as an absolute number. So the discernment stats for the above projects would be:
+### Actual Contribution
 
-| Project      | Discernment          |
+Your actual contribution is the average of your team's estimates of what percentage of the project you contributed to. It includes your own estimate of your contribution. Represented as a percentage (0-100%).
+
+In the above dataset, the player's actual contribution for the `#big-bees` project is the mean of _all_ the estimates, so it would be equal to 46.67% ( (50% + 42% + 48%) / 3 ).
+
+### Expected Contribution
+
+Your expected contribution how much you are expected to contribute to the project based on the team size so that each player contributes equally. Represented as a percentage (0-100%).
+
+In the above dataset, the player's expected contribution for the `#big-bees` project is equal to 33.33% ( 100% / 3 ) because their team has 3 players, so an equal contribution is 1/3 of the project.
+
+### Contribution Gap
+
+The contribution gap is the difference between actual and expected contribution. It demonstrates whether a player contributed _more_ or _less_ than is expected of them, based on the size of their team. Represented as a percentage (0-100%).
+
+In the above dataset, the player's contribution gap for the `#big-bees` project is equal to 13.34% ( 46.67% - 33.33% ).
+
+## Contribution Accuracy and Bias
+
+Contribution accuracy and bias reflect how well you estimate your contribution to projects.
+
+To demonstrate how these stats apply, we'll use a modified version of the above dataset, aggregating the team estimates into their average (mean):
+
+| Project      | Self estimate | Team estimate (avg) |
+|:-------------|:--------------|:--------------------|
+| #big-bees    | 50%           | 45%                 |
+| #red-rabbits | 28%           | 30%                 |
+| #tiny-tigers | 20%           | 30%                 |
+
+### Contribution Accuracy
+
+Contribution accuracy reflects how accurate your estimations are relative to the consensus. Represented as a percentage (0-100%).
+
+Zero is equal to perfect accuracy (i.e. you estimate your contribution the same as your peers do). The closer this stat is to zero, the more accurate your contribution estimates are.
+
+Your contribution accuracy _per project_ is the difference between self- and team-estimated contribution as an absolute number. So the contribution accuracy stats for the above projects would be:
```

Comment:
Put "absolute" in bold and link to what it means
http://www.coolmath.com/algebra/18-absolute-value-equations-inequalities/01-absolute-values-01